### DISCUSSION TITLE
Page just goes back to the top after processing the application

### DISCUSSION THREAD

**Discussion by flcorpuz** [community/community#70006] [GitHub Education] [answered]
### Hi everyone,

I am currently applying for the student developer pack. After the "Processing the application" loads the page goes back to the top and it does not show if there is any error or if my application has been submitted and pending. Is this a bug or does it mean my application is getting rejected?

**Comment by ks1686** [comment]
Having the same issue. Can't figure out who to contact.

**Reply by ks1686** [reply, marked as answer]
Found solution: it's returning an internal error you can't see. Something's wrong with the doc you're submitting, so either edit or submit a different doc.
